.github/workflows/vertexai.yml

```diff
@@ -15,6 +15,41 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check-mock-responses-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Clone mock responses
+      run: scripts/update_vertexai_responses.sh
+    - name: Find cloned and latest versions
+      run: |
+        echo "current_tag=$(git describe --tags | awk -F'/' '{print $NF}')" >> $GITHUB_ENV
```

Comment:
When running this command I'm getting 

`v0.8.0-47-g2bfe9e3`

in a misc repo. This happens when the latest commit isn't annotated with a tag.

This would make the equality check below never pass.

Reply:
I believe the commit that `git describe` sees will always be annotated with a tag since [update_vertexai_responses.sh](https://github.com/firebase/firebase-ios-sdk/pull/13206/files/9c3485a967fc6fa5fe9a80e178892503d5cfa9d5#diff-a3a9081de48dc326276b8bfe6b60f077e74af79784edbb11cfa972b0f72fd24e) clones a specific tag.

For example, the current latest commit in [vertexai-sdk-test-data](https://github.com/FirebaseExtended/vertexai-sdk-test-data) is not annotated with a tag, but cloning with
```
git clone --depth 1 --branch v1.0 https://github.com/FirebaseExtended/vertexai-sdk-test-data
```

and running `git describe --tags` shows `v1.0`.

Reply:
Then you only need `git tag -l` as there's only one tag available, right?